content/src/content/jcr_root/apps/core/wcm/components/page/v2/page/_cq_dialog/.content.xml

```diff
@@ -35,32 +35,53 @@
+                                    <templates
                                         jcr:primaryType="nt:unstructured"
-                                        jcr:title="AMP Configuration"
+                                        path="/mnt/overlay/wcm/foundation/components/basicpage/v1/basicpage/tabs/advanced/items/column/items/templates"
+                                        sling:resourceType="granite/ui/components/coral/foundation/include"/>
+                                    <authenticationrequirement
+                                        jcr:primaryType="nt:unstructured"
+                                        path="/mnt/overlay/wcm/foundation/components/basicpage/v1/basicpage/tabs/advanced/items/column/items/authenticationrequirement"
+                                        sling:resourceType="granite/ui/components/coral/foundation/include"/>
+                                    <export
+                                        jcr:primaryType="nt:unstructured"
+                                        path="/mnt/overlay/wcm/foundation/components/basicpage/v1/basicpage/tabs/advanced/items/column/items/export"
+                                        sling:resourceType="granite/ui/components/coral/foundation/include"/>
+                                    <amp
+                                        jcr:primaryType="nt:unstructured"
+                                        jcr:title="AMP"
                                         sling:resourceType="granite/ui/components/coral/foundation/form/fieldset">
                                         <items jcr:primaryType="nt:unstructured">
                                             <ampmode
                                                 jcr:primaryType="nt:unstructured"
                                                 sling:resourceType="granite/ui/components/coral/foundation/form/select"
                                                 composite="{Boolean}false"
-                                                fieldDescription="The mode in which you want AMP to be handled instead of what's defined in the page's content policy. Inherit from Page Template, default, will use the mode specified in the current template's page policy. Paired AMP will serve AMP pages when the 'amp' selector is present. No AMP will never serve AMP pages. AMP only will always serve AMP pages."
+                                                fieldDescription="The mode in which you want AMP to be handled instead of what's defined in the page's content policy. Paired AMP will serve AMP pages when the 'amp' selector is present. No AMP will never serve AMP pages. AMP only will always serve AMP pages."
                                                 fieldLabel="AMP Mode"
                                                 name="./ampMode">
                                                 <items jcr:primaryType="nt:unstructured">
```

Comment:
@isaacgibbs , is there a reason why we are not seeing the other modes here?
Users should be able to select "Inherit from template", "No AMP", "Paired AMP" or "AMP Only"

Reply:
@gaetanmarmasse I'm not sure what you mean? All of those options are defined in this field.

Reply:
Yep, sorry, I was seeing a weird diff.
I've changed minor things but it looked good! :)